Updated docs page (#1168)
dfinity/portal: docs/developer-docs/setup/cycles/cycles-faucet.md

You will need to install Internet Computer SDK following [this guide](/developer-docs/setup/install/index.mdx).

### Step 1: Get Coupon

First, you will need to navigate to <https://faucet.dfinity.org>. You

> will need to put in a request for cycles on the DFINITY DEV OFFICIAL Discord server. You can click on REQUEST CYCLES to join the discord server.

Reviewer: ```suggestion
will need to put in a request for cycles on the DFINITY DEV OFFICIAL Discord server. You can click on REQUEST CYCLES to join the Discord server.
```

![Getting Coupon](_attachments/faucet_step_1.png)

> You must complete a survey sent to you by domwoe as a Direct Message. Once completed domwoe will send you a coupon.

Reviewer: ```suggestion
You must complete a survey sent to you by someone from the DFINTIY team as a Direct Message. Once completed, they will send you a coupon.
```
I'd prefer not calling out specific people by name. Also, if written generically, this line will go out of date less often.

![Survey](_attachments/faucet_step_2.png)

Click NEXT STEP to continue.

### Step 2: Redeem Coupon

Once you get the coupon code, enter your coupon code.

![Enter Coupon](_attachments/faucet_step_3.png)

Click NEXT STEP to continue.

### Step 3: Setup SDK

You need to make sure your computer has dfx installed.

Also, run this command to check the version of dfx on your computer:

    dfx --version

If your dfx version is below 0.12.0, please run this command:

    sudo dfx upgrade

![Setup SDK](_attachments/faucet_step_4.png)

Click NEXT STEP to continue.

### Step 4: Claim Cycles

You will need to claim your free cycles by running this command:

![Claim Cycles](_attachments/faucet_step_5.png)

> The next step is to link the cycles wallet canister with your Principal Identifier. Run this command in your terminal:

Reviewer: ```suggestion
```
This step is no longer necessary now that I added `redeem-faucet-coupon`

>     dfx identify --network=ic set-wallet <canister id>

Reviewer: ```suggestion
```
